app/views/courses/_description_content.html.erb

```diff
@@ -0,0 +1,127 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h3 class="govuk-heading-m govuk-!-font-size-27">
+      <%= link_to 'About this course', manage_ui_course_page_url(provider_code: @provider[:provider_code], course_code: @course.attributes[:course_code]) + '/about', class: 'govuk-link' %>
+    </h3>
+    <dl class="govuk-summary-list govuk-summary-list--short">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          About this course
+        </dt>
+        <dd class="govuk-summary-list__value govuk-summary-list__value--truncate <%= 'course-parts__fields__value--empty' if @course.attributes[:about_course].nil? %>" data-qa="course__about">
+          <%= @course.attributes[:about_course] || 'Empty' %>
```

Comment:
Shall we add `&.present?` at the end of this, in case it's an empty string?

Reply:
Also, you shouldn't need to dive into `attributes` to pull this out. We had an issue in testing where the object being returned didn't quite behave like a resource, but iirs we added a `to_resource` method to sort this out.